lib/ditcher.js

```diff
@@ -482,6 +482,30 @@ Ditcher.prototype.doDeleteAll = function (params, callback) {
   });
 };
 
+Ditcher.prototype.doDropCollection = function (params, callback) {
+  var self = this;
+  if (!checkParams(params)) {
+    return callback(new Error("Invalid Params"));
+  }
+
+  if (params.type && params.type.length > MAX_COLLECTION_NAME) {
+    return callback("Error: 'type' name too long: '" + params.type + "'. Collection name cannot be greater than: " + MAX_COLLECTION_NAME);
+  }
+  var coll = constructCollectionName(params);
+
+  if (params.guid) {
+    return callback(new Error("Invalid Params - no guid required"));
+  }
+  self.logger.info("Ditcher.dropCollection/coll: " + coll);
```

Comment:
Using objects is nice because you can do `cat logfile | bunian -c 'this.collection == "name"'`, the API is actually the object first: `self.logger.info({collection: coll}, "Ditcher.dropCollection/coll");`